init.sh

```diff
@@ -43,7 +43,12 @@ if [[ -n "${marathon_host}" ]]; then
     echo "  ... starting Marathon Service Discovery on ${marathon_host}"
     touch /var/run/apigateway-config-watcher.lastrun
     # start marathon's service discovery
-    while true; do /etc/api-gateway/marathon-service-discovery.sh > /dev/stderr; sleep ${sleep_duration}; done &
+    #    init config
+    /etc/api-gateway/marathon-init.sh > /dev/stderr
```

Comment:
no it needs to be done before the the events/poll scripts are run, to prep the goji.conf.replaced file; It would be nice for goji to accept the marathon host as command line options, which would avoid this init.sh script step; i will log an issue there for that